server/src/testFixtures/java/io/crate/testing/TestingHelpers.java

```diff
@@ -419,7 +420,11 @@ public static <T> List<T> getRandomsOfType(int minLength, int maxLength, DataTyp
     }
 
     public static Map<String, Object> toMapping(BoundCreateTable boundCreateTable) {
-        // TODO: Add MetadataBuilder parameter to this method, so that analyzers test code and SQlExecutor.addTable can also assign oid.
+        // TODO: Assign oid in analyzer tests.
+        return toMapping(null, boundCreateTable);
```

Comment:
Assigning OID in unit tests is a long running TODO (see [comment](https://github.com/crate/crate/pull/14142#discussion_r1194874740)).

I plan to do it once the whole DROP COLUMN is done but here I had to partially do that since Indexer now asserts that targets have OIDS (depending on Version) and multiple tests are failing with that assertion.

To keep the scope smaller, I added OID only to SQlExecutor.addTable/addPartitioendTable - analyzer tests are still TODO